ruleset/rules/0515-exim_rules.xml

```diff
@@ -1,69 +1,69 @@
 <!--
-  -  Exim rules
-  -  Author: Alexandr Garaga.
-  -  Updated by Wazuh, Inc.
-  -  Copyright (C) 2015-2021, Wazuh Inc.
-  -  Copyright (C) 2009 Trend Micro Inc.
-  -  This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2.
+  Copyright (C) 2015-2021, Wazuh Inc.
+-->
+
+<!--
+  Exim rules 87500 - 87512
 -->
 
 <group name="exim,">
-    <rule id="87500" level="0">
-      <decoded_as>windows-date-format</decoded_as>
-      <regex>^\d\d\d\d-\d\d-\d\d \d\d:\d\d:\d\d SMTP </regex>
-      <description>Exim SMTP Messages Grouped.</description>
-    </rule>
+  <rule id="87500" level="0">
+    <decoded_as>windows-date-format</decoded_as>
+    <regex>^\d\d\d\d-\d\d-\d\d \d\d:\d\d:\d\d SMTP </regex>
+    <description>Exim SMTP Messages Grouped.</description>
+  </rule>
+
+  <rule id="87501" level="0">
+    <decoded_as>windows-date-format</decoded_as>
+    <regex>^\d\d\d\d-\d\d-\d\d \d\d:\d\d:\d\d dovecot</regex>
+    <description>dovecot messages grouped.</description>
+  </rule>
 
-    <rule id="87501" level="0">
-      <decoded_as>windows-date-format</decoded_as>
-      <regex>^\d\d\d\d-\d\d-\d\d \d\d:\d\d:\d\d dovecot</regex>
-      <description>dovecot messages grouped.</description>
-    </rule>
+  <rule id="87506" level="5">
+    <if_sid>87501</if_sid>
+    <match>authenticator failed</match>
+    <description>Exim Auth failed</description>
+    <group>invalid_login,authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,gdpr_IV_35.7.d,gdpr_IV_32.2,</group>
+  </rule>
 
-    <rule id="87506" level="5">
-      <if_sid>87501</if_sid>
-      <match>authenticator failed</match>
-      <description>Exim Auth failed</description>
-      <group>invalid_login,authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,gdpr_IV_35.7.d,gdpr_IV_32.2,</group>
-    </rule>
+  <rule id="87507" level="10" frequency="8" timeframe="240">
+    <if_matched_sid>87506</if_matched_sid>
+    <same_source_ip />
+    <description>Exim brute force attack (multiple auth failures).</description>
```

Comment:
Could we add descriptions like:
```log
Exim: Brute force attack (multiple auth failures).
```
Or is it better to leave as it is?

Reply:
Done